maven-core/src/main/java/org/apache/maven/toolchain/DefaultToolchainManagerPrivate.java

```diff
@@ -39,6 +43,19 @@
     extends DefaultToolchainManager
     implements ToolchainManagerPrivate
 {
+    @Inject
+    public DefaultToolchainManagerPrivate()
+    {
+        super();
+    }
+
+    /**
+     * Ctor for UT.
+     */
+    DefaultToolchainManagerPrivate( Logger logger )
+    {
+        super( logger );
```

Comment:
Same here...

Reply:
I did not touch UTs, I merely "cleaned up core logging". Still, added a TODO, as in default toolchain manager (and private) components there are some peculiarities: annotated component should not be extended (is bad practice), so IMO this needs some refactoring...